back/app-api-staging.yaml

```diff
@@ -0,0 +1,7 @@
+runtime: python38
+service: api-staging
+entrypoint: gunicorn -b :$PORT
+handlers:
+- url: /
```

Comment:
@vahidbazzaz Do you have an idea how to arrange it here that `v2/graphql` is not visible? Otherwise I'd have to add a dedicated Flask app, holding the 'query' endpoints only.
Or we ditch the idea of exposing the API on a subdomain `api*.` but instead use an URL suffix `/api`. Honestly that would simplify the CircleCI config again.

Reply:
@pylipp I concur with you and I think that absolutely make sense to have a dedicate end point for api, but we can check it out and see if that fine with other use cases (FE).

Reply:
We agreed that we want to use subdomains for exposing the query API.